packages/jest-environment-jsdom/src/index.ts

```diff
@@ -27,6 +27,7 @@ type Win = Window &
 
 class JSDOMEnvironment implements JestEnvironment {
   dom: JSDOM | null;
+  vm: DOMWindow | null;
```

Comment:
ditch this and do `this.dom.getInternalVMContext()` where it's needed instead. I'd guess that fixes the memory leak.

https://github.com/SimenB/jest-environment-jsdom-sixteen/blob/5cd4a4e28b934a57e16f07d6b24c0b8c393c7c55/index.js#L97-L109

Reply:
@lamhieu-vk modified in this [PR](https://github.com/lamhieu-vk/jest/pull/2). Thank you!

Reply:
thanks for commit, @irega